libs/coin-framework/src/derivation.ts

```diff
@@ -181,6 +181,9 @@ const modes: Readonly<Partial<Record<DerivationMode, unknown>>> = Object.freeze(
   internet_computer: {
     overridesDerivation: "44'/223'/0'/0/<account>",
   },
+  kadena: {
+    overridesDerivation: "44'/223'/<account>'/0/0",
+  },
```

Comment:
Are we sure about this path ? 
I have 626 on SLIP44 (https://github.com/satoshilabs/slips/blob/master/slip-0044.md)

Reply:
It's definitely wrong! Looks like copied from the above lines.